service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java

```diff
@@ -198,6 +313,8 @@ public GetAllTeamsResponse getAllTeams(JWTData userData) {
             .from(TEAM)
             .innerJoin(USER_TEAM)
             .on(TEAM.ID.eq(USER_TEAM.TEAM_ID))
+            .where(USER_TEAM.TEAM_ROLE.eq(TeamRole.LEADER)
```

Comment:
Shouldn't getAllTeams still return all teams, not just the users? Because it's the lightweight response without member data?